cloud-deployment-guide/editor-plugin-version.md

```diff
@@ -13,7 +13,7 @@ Use the URL provided to specify the TinyMCE version when deploying via Tiny Clou
 The following example is the default for loading TinyMCE 5.0 via Tiny Cloud:
 
 ```js
```

Comment:
We probably need to explain no-api-key at least once on this page.

Reply:
done:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/